src/Services/HtmlConverter/HtmlConverterInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Tychovbh\Mvc\Services\HtmlConverter;
+
+interface HtmlConverterInterface
+{
+    /**
+     * @param string $html
+     * @return mixed
```

Comment:
this should return $this, so $return HtmlConverterInterface
als add this to the method page(): HtmlConverterInterface;